### DISCUSSION TITLE
Welcome to py-discord-sdk Discussions!

### DISCUSSION THREAD

**Discussion by LennyPhoenix** [LennyPhoenix/py-discord-sdk#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RoARene317** [comment]
Hello, for documentation , I recommend to put to https://readthedocs.org/ instead here. It much easier.

**Reply by LennyPhoenix** [reply]
Someone was working on readthedocs support, but I had trouble getting it to work. I will try and fix it at some point soon.
